BrainPortal/app/views/bourreaux/show.html.erb

```diff
@@ -104,22 +104,21 @@
 
     <% if is_portal %>
 
-      <% t.edit_cell(:help_url, :header => "User Manual URL", :show_width => 1) do %>
-        <%= text_field_tag "bourreau[help_url]", @bourreau.help_url, :size => 40 %><br/>
-        <div class="field_explanation">If set, the portal will show a link called 'User Manual' in the account bar at the top.</div>
-      <% end %>
-
-      <% t.edit_cell 'meta[upload_size_limit]',
-                     :header => "File upload size limit (MB)",
-                     :content => (@bourreau.meta[:upload_size_limit].blank? ? "(None)" : @bourreau.meta[:upload_size_limit]),
+      <% t.edit_cell :site_url_prefix,
```

Comment:
Because there was no way before to set the +site_url_prefix+, which is necessary for the email to the person registering to be able to find the site.